roc/types.go

```diff
@@ -0,0 +1,91 @@
+// WARNING: This file has automatically been generated on Sun, 13 Oct 2019 20:21:32 CEST.
+// Code generated by https://git.io/c-for-go. DO NOT EDIT.
+
+package roc
+
+/*
+#cgo linux CFLAGS: -I"../include"
+#cgo linux LDFLAGS: -lroc
+#include "../include/roc/address.h"
+#include "../include/roc/config.h"
+#include "../include/roc/context.h"
+#include "../include/roc/frame.h"
+#include "../include/roc/log.h"
+#include "../include/roc/platform.h"
+#include "../include/roc/receiver.h"
+#include "../include/roc/sender.h"
+#include <stdlib.h>
+#include "cgo_helpers.h"
+*/
+import "C"
+import "unsafe"
+
+// Address as declared in roc/address.h:59
+type Address struct {
+	ref94efe9d7    *C.roc_address
+	allocs94efe9d7 interface{}
+}
+
+// ContextConfig as declared in roc/config.h:147
+type ContextConfig struct {
+	maxPacketSize  uint32
+	maxFrameSize   uint32
```

Comment:
All fields in ContextConfig, SenderConfig, and ReceiverConfig are unexported. We should export them I guess?

Reply:
yes, I think I can switch it on/off with c-for-go, but my attempts were unsuccessful so far...

Reply:
done

Reply:
Forgot to push?

Reply:
Forgot to push?

Reply:
yup.